src/FakeItEasy/Core/FakeManager.EventRule.cs

```diff
@@ -120,7 +120,7 @@ private void RaiseEvent(EventCall call, IEventRaiserArgumentProvider argumentPro
             {
                 if (this.RegisteredEventHandlers.TryGetValue(call.Event, out var raiseMethod))
                 {
-                    var arguments = argumentProvider.GetEventArguments(this.fakeManager.Object);
+                    var arguments = argumentProvider.GetEventArguments(this.fakeManager.Object!);

```

Comment:
This code runs when we use the `+= Raise...` syntax, so the calling code must have a strong reference to the fake object (proxy). So there's no chance that the fake could have been collected, so FakeManager.Object won't return null.